.github/workflows/pr-build.yml

```diff
@@ -40,4 +40,20 @@ jobs:
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
         with: 
-          args: build -x avroCompatibilityCheck dockerBuildImages
\ No newline at end of file
+          args: build -x avroCompatibilityCheck dockerBuildImages
```

Comment:
I think we are publishing the avro classes, so avroCompatibility check will pass.